### PR TITLE
Add Renfe information in spanish stations

### PR DESCRIPTION
**Pull request by paulRbr**
This MR adds for all spanish stations that we know:
- a `renfe_id` value
- a `info:es` value
- possibly a `db_id` value

### CODE REVIEW

**Comment by mckbrmn** [comment]
Usage is to abstain from repeating information that's in the station name in the info sections. If the station name is incorrect, please fix the name and slug. If it's only additional information, please look into adding it in other languages.

File: stations.csv [hunk 1]
```diff
@@ -4991,86 +4991,86 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
 6401;Zürich;zurich;;;47.378968;8.540534;;t;CH;f;Europe/Zurich;f;CHZRH;;t;;f;8596008;t;;f;;f;;f;;f;;f;;;f;;;;;;;;;;;;;;;;;;
 6402;;;;;;;;f;CH;f;Europe/Zurich;f;;;f;;f;;f;;f;;f;;f;;f;;f;;;f;;;;;;;;;;;;;;;;;;
 6403;Interlaken;interlaken;;;;;;t;CH;f;Europe/Zurich;f;CHZTJ;;f;;f;;f;;f;;f;;f;;f;;f;;;f;;;;;;;;;;;;;;;;;;
-6404;Tortosa;tortosa;7165400;;;;;f;ES;f;Europe/Madrid;f;ESAAA;;f;;f;;f;;f;;f;;f;;f;;f;;;f;;;;;;;;;;;;;;;;;;
+6404;Tortosa;tortosa;7165400;;;;;f;ES;f;Europe/Madrid;t;ESAAA;;f;;f;7100121;f;;f;;f;;f;;f;;f;65400;t;f;;;;Tortosa;;;;;;;;;;;;;;
```
Comment (Tristramg) [inline stations.csv]: Tortosa ! :heart:

File: stations.csv [hunk 2]
```diff
@@ -5079,205 +5079,205 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
...
-6675;Pamplona;pamplona;7180100;;41.378911;2.140367;;f;ES;f;Europe/Madrid;f;ESPNA;;f;;f;7100080;f;;f;;f;;f;;f;;f;;;f;;;;;;;;;;;;;;;;;;
+6667;Madrid Atocha;madrid-atocha;7160000;;40.405963;-3.689757;6663;f;ES;f;Europe/Madrid;t;ESMAT;;t;;f;7100049;f;;f;;f;;f;;f;;f;60000;t;f;;;;Madrid-Puerta De Atocha;;;;;;;;;;;;;;
+6668;Madrid Chamartín;madrid-chamartin;7117000;;40.472151;-3.68235;6663;f;ES;t;Europe/Madrid;t;ESMCH;;f;;f;7100017;f;;f;;f;;f;;f;;f;17000;t;f;;;;Madrid-Chamartin;;;;;;;;;;;;;;
+6669;Medina del Campo;medina-del-campo;7110500;;;;;f;ES;f;Europe/Madrid;t;ESMDC;;f;;f;7100002;f;;f;;f;;f;;f;;f;10500;t;f;;;;Medina Del Campo;;;;;;;;;;;;;;
+6670;Miranda de Ebro;miranda-de-ebro;7111200;;;;;f;ES;f;Europe/Madrid;t;ESMEB;;f;;f;7100006;f;;f;;f;;f;;f;;f;11200;t;f;;;;Miranda De Ebro;;;;;;;;;;;;;;
+6671;Murcia del Carmen;murcia-del-carmen;7161200;;;;;f;ES;f;Europe/Madrid;t;ESMJV;;f;;f;7100052;f;;f;;f;;f;;f;;f;61200;t;f;;;;Murcia;;;;;;;;;;;;;;
+6672;Cordoue;cordoue;7150500;;;;;f;ES;f;Europe/Madrid;t;ESODB;;f;;f;7100038;f;;f;;f;;f;;f;;f;50500;t;f;;;;Cordoba;;;;;;;;;;;;;;
```
Comment (vdegove-ct) [inline stations.csv]: Cordoba.

File: stations.csv [hunk 3]
```diff
@@ -5079,205 +5079,205 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
...
+6682;Salamanca;salamanca;7130100;;41.133002;0.859998;;f;ES;f;Europe/Madrid;t;ESSLM;;f;;f;7100027;f;;f;;f;;f;;f;;f;30100;t;f;;;;Salamanca;;;;;;;;;;;;;;
 6683;Sevilla;sevilla;;;;;;t;ES;f;Europe/Madrid;f;ESSVQ;;f;;f;;f;;f;;f;;f;;f;;f;;;f;;;;;;;;;;;;;;;;;;
-6684;Camp Tarragona;camp-tarragona;7104104;;41.19206141706671;1.272541880607605;;f;ES;f;Europe/Madrid;t;ESTAA;;t;;f;;f;;f;;f;;f;;f;;f;;;f;;;;;;;;;;;;;;;;;;
-6685;Torremolinos;torremolinos;7154509;;;;;f;ES;f;Europe/Madrid;f;ESTRM;;f;;f;;f;;f;;f;;f;;f;;f;;;f;;;;;;;;;;;;;;;;;;
-6686;Tuy;tuy;7122401;;;;;f;ES;f;Europe/Madrid;f;ESTUY;;f;;f;;f;;f;;f;;f;;f;;f;;;f;;;;;;;;;;;;;;;;;;
+6684;Camp Tarragona;camp-tarragona;7104104;;41.19206141706671;1.272541880607605;;f;ES;f;Europe/Madrid;t;ESTAA;;t;;f;;f;;f;;f;;f;;f;;f;04104;t;f;;;;Camp Tarragona;;;;;;;;;;;;;;
+6685;Torremolinos;torremolinos;7154509;;;;;f;ES;f;Europe/Madrid;t;ESTRM;;f;;f;;f;;f;;f;;f;;f;;f;54509;t;f;;;;Torremolinos;;;;;;;;;;;;;;
+6686;Tuy;tuy;7122401;;;;;f;ES;f;Europe/Madrid;t;ESTUY;;f;;f;;f;;f;;f;;f;;f;;f;22401;t;f;;;;Tui /Tuy;;;;;;;;;;;;;;
```
Comment (vdegove-ct) [inline stations.csv]: In this kind of cases, I'd put just Tui in the info_es field. For now, "Tuy (Tui / Tuy)" will be displayed in the apps, there is a repetition.

File: stations.csv [hunk 4]
```diff
@@ -5079,205 +5079,205 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
...
 6687;Valencia de Alcantar;valencia-de-alcantar;;;;;;t;ES;f;Europe/Madrid;f;ESVAL;;f;;f;;f;;f;;f;;f;;f;;f;;;f;;;;;;;;;;;;;;;;;;
-6688;Venta de Banos;venta-de-banos;7111000;;;;;f;ES;f;Europe/Madrid;f;ESVDB;;f;;f;;f;;f;;f;;f;;f;;f;;;f;;;;;;;;;;;;;;;;;;
+6688;Venta de Banos;venta-de-banos;7111000;;;;;f;ES;f;Europe/Madrid;t;ESVDB;;f;;f;;f;;f;;f;;f;;f;;f;11000;t;f;;;;Venta De Baños;;;;;;;;;;;;;;
 6689;Vigo;vigo;7122303;;;;;f;ES;f;Europe/Madrid;f;ESVGO;;f;;f;;f;;f;;f;;f;;f;;f;;;f;;;;;;;;;;;;;;;;;;
-6690;Valence (Espagne);valence-espagne;7165000;;;;;f;ES;f;Europe/Madrid;f;ESVLC;;f;;f;;f;;f;;f;;f;;f;;f;;;f;;;;;;;;;;;;;;;;;;
+6690;Valence (Espagne);valence-espagne;7165000;;;;;f;ES;f;Europe/Madrid;t;ESVLC;;f;;f;7100054;f;;f;;f;;f;;f;;f;65000;t;f;;;;Valencia-Estacio Del Nord;;;;;;;;;;;;;;
```
Comment (vdegove-ct) [inline stations.csv]: "Valencia" for the main name and ID, and "Valence – Espagne" for the info_fr ?

File: stations.csv (hunk 2 above)
Comment (vdegove-ct) [inline stations.csv]: And "Cordoue" for info_fr

**Comment by paulRbr** [comment]
This MR will be superseded by another checkup by @Tristramg

**Pull request closed without merging**